100of100Service/src/main/java/dev/coms4156/project/DatabaseConnection.java

```diff
@@ -9,26 +11,103 @@
  */
 public class DatabaseConnection {
   private volatile static DatabaseConnection instance;
+  private Connection connection;
 
-  private DatabaseConnection() {
-    // TODO: Initialize the MySQL database connection, maybe ip, user name, and password here?
+  protected DatabaseConnection() {
+    try {
+      String url = "jdbc:mysql://database-100-team.c7mqy28ys9uq.us-east-1.rds.amazonaws.com:3306/organization_management";
+      String user = "admin";
+      String password = "sxy6cJEmv6iLT61qs7DO";
+      this.connection = DriverManager.getConnection(url, user, password);
+    } catch (SQLException e) {
+      e.printStackTrace();
+    }
   }
 
-  public List<Employee> getEmployees(int clientId) {
-    // TODO: Maybe some basic SQL query?
-    return null;
+  public List<Employee> getEmployees(int organizationId) {
+    List<Employee> employees = new ArrayList<>();
+    String query = "SELECT * FROM employees WHERE organization_id = ?";
+    try (PreparedStatement pstmt = connection.prepareStatement(query)) {
+      pstmt.setInt(1, organizationId);
+      ResultSet rs = pstmt.executeQuery();
+      while (rs.next()) {
+        Employee employee = new Employee(
+                null, // HRDatabaseFacade instance, passing null for now
```

Comment:
Need to think about a way to solve this. Otherwise, the Object may not be able to call back.